facility-manager.html

```diff
@@ -0,0 +1,806 @@
+
+<p>OMERO point releases i.e. 5.2.x occur approximately quarterly, and breaking releases i.e. 5.x roughly bi-annually. Clients and servers are backwardly compatible for point releases, but not for breaking releases.</p>
+
+<p>Given the number of server operating systems, clients and components involved in OMERO, the matrix of versions and compatibility is large. If you are in any doubt about the suitability for upgrade of your particular installation, draw on the wide experience of the OMERO community via the <a href="http://www.openmicroscopy.org/community/" target="_blank" class="external-url">OME Forums</a> or <a href="http://www.openmicroscopy.org/site/community/mailing-lists" target="_blank" class="external-url">OME Mailing Lists</a>, giving as much detail as you can.</p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<h2><a class="anchor" id="publish"></a>Using OMERO for publication</h2>
+
+<p>Public groups can be used to make data accessible without the need for authentication. The permissions on the public group are usually set to Read-Only. In order for a user to be able to place data in the public group, they must ask the Administrator to make them a member of the group. Details on how to configure public groups and make them publically accessible on the OMERO server are in the <a href="https://www.openmicroscopy.org/site/support/omero5.2/sysadmins/public.html#public-user" target="_blank" class="external-url">Public user</a> section of the system administrator documentation.</p>
+
+<p>As data in a public group is accessible without login credentials, this enables image thumbnails, from the OMERO server, and links to open images in the full OMERO.web viewer, to be embedded in any HTML-based page. Details of how to embed thumbnails and links in web pages or Wikis are in the User Help <a href="http://help.openmicroscopy.org/publish.html" target="_blank" class="external-url">Publishing with OMERO</a> section.</p>
+
+<p>The full OMERO.web viewer can be embedded in pages as a customisable OMERO viewport. The level of interactivity can be set by adding buttons, links or scalebars etc. as desired. Details on the viewport can be seen in <a href="https://www.openmicroscopy.org/site/support/omero5.2/developers/Web/ViewPort.html#embedding-web-viewport" target="_blank" class="external-url">Customizing the content of the embedded OMERO.web viewport</a>.</p>
+
+<p>OMERO is widely used as the basis for public data repositories. OMERO.gallery provides a basic gallery front end based on the Group-Project-Dataset-Image, and is a good starting point to develop a bespoke front end. Full documentation on how to set up OMERO as a public repository and details of customisation options can be found in the OMERO Repository documentation.</p>
```

Comment:
repository documentation at the end should link to https://www.openmicroscopy.org/site/support/omero/sysadmins/index.html#optimizing-omero-as-a-data-repository